### ISSUE TITLE
Should the A-case be using stub land?

### ISSUE THREAD

**Issue by rljacob** [ESMCI/cime#1091]
An A-case is supposed to include all the main data models together.  But the current longname for the A-case has stub land:

2000_DATM%NYF_SLND_DICE%SSMI_DOCN%DOM_DROF%NYF_SGLC_SWAV'

Is used to be DLND%NULL.  Was that switch intentional?

**Comment by jedwards4b** [comment]
I don't think so - looking now.   Why would testing have missed this?

**Comment by jedwards4b** [comment]
It seems to be intentional - the A compset is defined in driver/cime_config/config_compsets.xml as
```  <compset>
    <alias>A</alias>
    <lname>2000_DATM%NYF_SLND_DICE%SSMI_DOCN%DOM_DROF%NYF_SGLC_SWAV</lname>
  </compset>```   git blame shows this line last changed by @mvertens in September.

**Comment by rljacob** [comment]
@mvertens can you explain why you made that change and close this issue?

**Comment by billsacks** [comment]
I'm pretty sure this was resolved by #1365

**Comment by billsacks** [comment]
Looking back at this issue, I'm not positive why the A compset uses SLND. But my thinking is that it has to do with what grids are supported for different data models: dlnd in "snow" mode has only supported f09_g16, I think, whereas I'm guessing that we often test the other data models with different grids than that.
